semgrep-core/parsing/Parse_kotlin_tree_sitter.ml

```diff
@@ -465,8 +465,9 @@ and annotation (env : env) (x : CST.annotation) =
 and anon_choice_param_b77c1d8 (env : env) (x : CST.anon_choice_param_b77c1d8) =
   (match x with
    | `Param x -> parameter env x
-   | `Type x -> let _ =  type_ env x in
-       raise Todo
+   | `Type x ->
+       let v1 =  type_ env x in
+       ParamPattern (PatType v1)
```

Comment:
I tyink you can build a ParamClassic with p_type = v1, and p_name = None,
like we do in C for function signatures.

Reply:
There is even AST_generic.param_of_type() to help build this parameter_classic type.